src/components/pages/CaseDetails/RenderCaseDetails.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Layout, Card } from 'antd';
+import NavHeader from '../../common/NavHeader';
+import BreadCrumbs from '../../common/BreadCrumbs';
+import { CaseDetailsCollapse } from '../../common/CaseDetailsCollapse';
+const { Header, Content, Footer } = Layout;
+
+const RenderCaseDetails = () => {
+  const location = useLocation();
+
+  return (
+    <div className="RenderLayoutTemplate">
+      <Layout className="Layout">
+        <Header className="Header">
+          <NavHeader />
+        </Header>
+        <Content className="Content">
+          <BreadCrumbs pathname={location.pathname} />
```

Comment:
From the looks of it, the `BreadCrumbs ` component requires us to manually add a url into `breadCrumbNameMap` to be displayed. Since case-view is dynamic this may cause some issues. How would you recommend fixing this?

Reply:
Good point! I'm working on a fix so that if `breadCrumbNameMap[pathname]` is falsy a function runs that returns a string template literal with a formatted dynamic pathname. Commit incoming shortly for review

Reply:
Awesome